ProcessOptimizer/tests/test_optimizer.py

```diff
@@ -328,3 +332,60 @@ def test_iterating_ask_tell_lhs():
     assert opt.ask(n_points=3) == samples[2:5]  # samples 2,3 and 4
     opt.tell([[2], [2], [2]], [0, 0, 0])
     assert opt.ask() == samples[5]  # sample 5
+
+
+@pytest.mark.slow_test
+def test_add_remove_modelled_noise():
+    '''
+    Tests whether the addition of white noise leads to predictions closer to
+    known true values of experimental noise (iid gaussian noise)'''
+    # Define objective function
+    def flat_score(x):
+        return 42
+    
+    # Set noise and model system
+    noise_size = 0.45
+    flat_space = [(-1.0, 1.0)]
+    flat_noise = {"model_type": "constant",
+                  "noise_size": noise_size,
+                  }
+    # Build ModelSystem object
+    model = ModelSystem(score=flat_score,
+                        space=flat_space,
+                        noise_model=flat_noise,
+                        )
+    # Instantiate Optimizer
```

Comment:
This is real nit-picky stuff, but across the code-base we mainly format multi-line function input in this way:
```python
model = ModelSystem(
    score=flat_score,
    space=flat_space,
    noise_model=flat_noise,
)
```
That is, new line after starting ( then indent by one tab, each argument on its own line, closing ) on its own line at an indentation level equal to that of the starting line.

Reply:
Will fix